bin/bootstrap

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
Okie dokie! I was just putting everything in `bin` since that is where setup was as well. But sounds good to me!